common/middleware/errorhandler.go

```diff
@@ -78,3 +81,14 @@ func (i *errorInterceptor) Write(data []byte) (int, error) {
 	}
 	return len(data), nil
 }
+
+// errorInterceptor also implements net.Hijacker, to let the downstream Handler
+// hijack the connection. This is needed by the app-mapper's proxy.
```

Comment:
I would remove the `app-mapper` reference since it doesn't formally exist anymore (and it's unkown to OSS devs)